<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>history_menu</class>
 <widget class="QDialog" name="history_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="history_menu_scrollarea">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="history_menu_scrollarea_content">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>399</width>
      <height>249</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="history_menu_back">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
